Makefile

```diff
@@ -105,40 +107,30 @@ proxy: $(shell find $(proxy) $(find_options))
 wallet-prod: wallet-node-modules $(shell find $(wallet)/src $(find_options))
 	$(log_start)
 	$(docker_run_in_wallet) "rm -f .env && cp ops/prod.env .env"
-	$(docker_run_in_wallet) "ln -sf /client node_modules/connext"
-	$(docker_run_in_wallet) "yarn build"
+	$(docker_run_in_wallet) "npm run build"
 	$(log_finish) && touch build/wallet-prod
 
 wallet: wallet-node-modules $(shell find $(wallet)/src $(find_options))
 	$(log_start)
 	docker build --file $(wallet)/ops/dev.dockerfile --tag $(project)_wallet:dev $(wallet)
 	$(log_finish) && touch build/wallet
 
-wallet-node-modules: $(project)_builder $(wallet)/package.json client
+wallet-node-modules: builder client $(wallet)/package.json
 	$(log_start)
-	$(docker_run_in_wallet) "yarn install --network-timeout 1000000"
+	$(docker_run_in_wallet) "rm -f node_modules/connext"
```

Comment:
I switched it back because cleaning worked and I didn't want to make any unnecessary downstream issues. Please push a fix if you think it should be a different way.